plenum/server/node.py

```diff
@@ -2411,13 +2422,109 @@ def doStaticValidation(self, request: Request):
 
         self.execute_hook(NodeHooks.POST_STATIC_VALIDATION, request=request)
 
+    def validateTaaAcceptance(self, request: Request):
+        ledger_id = self.ledger_id_for_request(request)
+
+        if not self.ledgerManager.ledger_info(ledger_id).taa_acceptance_required:
+            logger.trace(
+                "{} TAA acceptance passed for request {}: "
+                "not required for ledger id {}"
+                .format(self, request.reqId, ledger_id)
+            )
+            return
+
+        config_req_handler = self.ledger_to_req_handler.get(CONFIG_LEDGER_ID)
```

Comment:
I think it's better to use `get_req_handler(ledger_id=CONFIG_LEDGER_ID)`

Reply:
agreed, updated